src/System.Linq.Expressions/src/System/Runtime/CompilerServices/ReadOnlyCollectionBuilder.cs

```diff
@@ -495,7 +495,7 @@ public ReadOnlyCollection<T> ToReadOnlyCollection()
         /// </summary>
         /// <param name = "min">The minimum value to have allocated.</param>
         /// <remarks>
-        /// If the current capacity of the list is less than <paramref name = "min" />,
+        /// If the current capacity of the list is less than <paramref name = "min"/>,
```

Comment:
@AlexRadch I did not catch this before, but there should also not be spaces surrounding the equals. `<param name="min">` above and `<paramref name="min"/>` Sorry about that :smile: